encoding/resampling.md

```diff
@@ -624,6 +624,6 @@ shifted_scaled_u = core.resize.Spline16(u, 1920, 1080, src_left=0.25) # shifts t
 [^1]: The Fourier transform is an ubiqitous concept in image processing, so we strongly advise becoming familiar with at least the basics. A very good resource for this topic is [ImageMagick’s guide][].
 [^2]: Robidoux, N. (2012, October 21). Resampling — ImageMagick v6 Examples. Retrieved August 22, 2019, from https://www.imagemagick.org/Usage/filter/nicolas/#upsampling
 [^3]: If you don’t understand what this means, read the resources linked above in the [resizing section](#resizing).
-[^4]: This is derived as follows: The shift is the distance between the position of the first luma sample and the position of the first chroma sample (both mapped onto the input grid and given in terms of input chroma pixel widths). The former is located at `0.25 + (src_width/dst_width)/4`, the latter at `(src_width/dst_width)/2`. This yields `0.25 + (src_width/dst_width)/4 - (src_width/dst_width)/2 = 0.25 + (src_width/dst_width) * (1/4 - 1/2) = 0.25 + (src_width/dst_width) * -0.25` for the shift.
+[^4]: This is derived as follows: The shift is the distance between the position of the first luma sample and the position of the first chroma sample (both mapped onto the input grid and given in terms of input chroma pixel widths). The former is located at $$\left( 0.25 + \frac{\mathrm{src~width}}{4 \times \mathrm{dst~width}} \right)$$, the latter at $$\left( \frac{\mathrm{src~width}}{2 \times \mathrm{dst~width}} \right)$$. This yields $$\Big[ 0.25 + \frac{\mathrm{src~width}}{4 \times \mathrm{dst~width}} - \frac{\mathrm{src~width}}{2 \times \mathrm{dst~width}} = 0.25 + \frac{\mathrm{src~width}}{\mathrm{dst~width}} \times (^1/_4 -\,^1/_2) = 0.25 + \frac{\mathrm{src~width}}{\mathrm{dst~width}} \times (-0.25) \Big]$$ for the shift.
```

Comment:
Again, why `\Big`? Also sure would be nice if footnotes could have line breaks. 😞

Reply:
Similar to the other response, it helps separate it (also good for line height when it's placed below due to sizing changes) from the text.
#### Big
![image](https://user-images.githubusercontent.com/10972040/66612185-f3d1b100-eb8e-11e9-8305-2ad66a3a9dcc.png)
![image](https://user-images.githubusercontent.com/10972040/66612195-fa602880-eb8e-11e9-8072-9df310687f70.png)

#### normal `[ ]`
![image](https://user-images.githubusercontent.com/10972040/66612248-31ced500-eb8f-11e9-94bc-c038d7d0b6b6.png)
![image](https://user-images.githubusercontent.com/10972040/66612275-3e532d80-eb8f-11e9-8848-1c22fb00d707.png)

#### old
![image](https://user-images.githubusercontent.com/10972040/66612285-4c08b300-eb8f-11e9-9b98-3242e7782afb.png)
![image](https://user-images.githubusercontent.com/10972040/66612294-50cd6700-eb8f-11e9-9f78-ac2a4c95b763.png)

Reply:
Could maybe even use an non-breaking space inside on both sides

Reply:
Hm, with a long formula like this, we should probably try looking into using an info box instead of a footnote. It's still extra information that's not necessary to grasp the content, but a footnote probably isn't the best place for ti.